model/src/guild/audit_log/event_type.rs

```diff
@@ -66,15 +66,15 @@ pub enum AuditLogEventType {
     /// [Member]: super::super::Member
     /// [role]: super::super::Role
     MemberRoleUpdate = 25,
-    /// [Member] was moved to a [voice channel].
+    /// [Member] was moved between audio [channel]s.
```

Comment:
Call it voice for consistency.

```suggestion
    /// [Member] was moved between voice [channel]s.
```

Reply:
i think it is implied that this may change in the future

Reply:
Shouldn't this be changed in a separate PR then?